assets/src/application/ui/input/Button/style.scss

```diff
@@ -62,14 +62,15 @@
 		display: inline-flex;
 		height: var(--ee-icon-button-size);
 		line-height: var(--ee-font-size-default);
-		margin-bottom: var(--ee-margin-smaller);
+		margin: var(--ee-margin-smaller);
 		outline: none;
 		padding: var(--ee-padding-tiny) var(--ee-padding-smaller);
 		position: relative;
 		text-decoration: none;
 		text-align: center;
 		text-shadow: var(--ee-text-shadow-inset-light);
-		transition: background-color 50ms border-color 50ms ease 50ms, color 50ms linear 50ms;
+		-webkit-transition: all linear 50ms;
```

Comment:
seems like this prefix is optional:
![image](https://user-images.githubusercontent.com/1477453/84487434-6c04ef00-aca7-11ea-9efb-c83d9372cf32.png)
***
http://shouldiprefix.com/#transitions

Reply:
that was only added to override that same selector rule from WP core CSS
I was just trying to get things fixed to my liking, but you're 150% correct that this could have been handled better, most likely by increasing the specificity of the rule. The issue was that the WP core CSS was using a rule like `.wp-core-ui select { //styles }` where the `wp-core-ui` class was applied to a high level container that wrapped ALL of our code and therefore applied to everything we were doing.

anyways, not trying to make excuses, just explaining why it was done, i know i should feel bad for this :(

Reply:
I wasn't aware of `.wp-core-ui select` issue, sorry for my ignorance